File: src/main/java/com/market/ddanggyeo/user/service/UserInfoService.java
package com.market.ddanggyeo.user.service;

import com.market.ddanggyeo.user.dto.UserDto;
import org.springframework.security.core.Authentication;
import org.springframework.security.core.context.SecurityContextHolder;
import org.springframework.stereotype.Service;

@Service
public class UserInfoService {

    public String insertUserInfo(UserDto userDto) {
        Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
        return "";
    }
}

File: src/main/kotlin/com/market/ddanggyeo/user/service/UserInfoService.kt
package com.market.ddanggyeo.user.service

import com.market.ddanggyeo.user.dto.UserDto
import org.springframework.security.core.context.SecurityContextHolder
import org.springframework.stereotype.Service

@Service
class UserInfoService() {

    fun insertUserInfo(userDto: UserDto): String {
        val authentication = SecurityContextHolder.getContext().authentication

        return ""
    }
}